### DISCUSSION TITLE
math: `\operatorname` not working

### DISCUSSION THREAD

**Discussion by nschloe** [community/community#55368] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

````markdown
```math
\operatorname*{Min}_{x}
```
````

> ```math
> \operatorname*{Min}_{x}
> ```

**Comment by Jordan99990** [comment]
\usepackage{amsmath}
...
\[
\operatorname*{Min}_{x} f(x),
\]

use it like this in your latex document

**Reply by nschloe** [reply]
This is about the Markdown renderer...

**Comment by 3f6a** [comment]
Ref.: https://github.com/github/markup/issues/1688
